independent-publisher-connectors/The Meal DB/TheMealDB.swagger.json

```diff
@@ -0,0 +1,141 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TheMealDB",
+    "description": "Its an open online database of food meal recipes. \nRecipe books are old school, there must be a better way!\n- High quality transparent PNG ingredients\n- Meal image uploads\n- Browse meals in sequence\n- Area and Categories",
+    "version": "1.0"
+  },
+  "host": "themealdb.com",
+  "basePath": "/api/json/v1/1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/search.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Search Meal",
+        "description": "Search meal by name. You can also search by the first single letter.",
+        "operationId": "SearchMeal",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "s",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      }
+    },
+    "/lookup.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Look for a Meal with id",
+        "description": "Lookup full meal details by id",
+        "operationId": "SearchMealByID",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "i",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      }
+    },
+    "/random.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get a random meal",
+        "description": "Lookup a random meal",
+        "operationId": "GetARandomMeal",
+        "x-ms-visibility": "important",
+        "parameters": []
+      }
+    },
+    "/list.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "List all Areas",
+        "description": "List all Areas where meals are mostly taken",
+        "operationId": "ListAreas",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "a",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      }
+    },
+    "/filter.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Filter meals by ingredient",
+        "description": "Filter by main ingredient",
```

Comment:
@johnmuchiriM, all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)